.jenkins/lsu-perftests/entry.sh

```
@@ -59,9 +59,36 @@ status_file="jenkins-hpx-${configuration_name}-ctest-status.txt"
 
 # Comment on the PR if any failures
 if [[ $(cat ${status_file}) != 0 ]]; then
-    ./.jenkins/lsu-perftests/comment_github.sh
+    github_commit_status="failure"
+else
+    github_commit_status="success"
 fi
 
+cdash_build_id="$(cat jenkins-hpx-${configuration_name}-cdash-build-id.txt)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)